### PR TITLE
LPS-162618 | master

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the fix for [LPS-162618](https://issues.liferay.com/browse/LPS-162618), the original has passed my review at sontruongces#33. Please help me to review and give feedback.

Thanks!
cc: @ces-huynguyen

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance.jsp [hunk 1]
```diff
@@ -109,6 +110,9 @@ FormInstancePermissionCheckerHelper formInstancePermissionCheckerHelper = ddmFor
 											if (!hasValidDDMFormFields) {
 												errorMessage = LanguageUtil.format(request, "this-form-was-created-using-a-custom-field-type-x-that-is-not-available-for-this-liferay-dxp-installation.-instal-x-to-make-it-available-for-editing", ddmFormAdminDisplayContext.getInvalidDDMFormFieldType(formInstance));
 											}
+											else if (!isNotDeletedObject) {
+												errorMessage = "this-form-was-created-using-a-deleted-object-as-storage-type";
+											}
 											else if (!hasValidMappedObject) {
 												errorMessage = LanguageUtil.format(request, "this-form-was-created-using-an-inactive-object-as-storage-type.-activate-x-object-to-make-it-available-for-editing", ddmFormAdminDisplayContext.getObjectDefinitionLabel(formInstance, locale));
 											}
```
[the remark this answers is not in the document]
Comment (ces-huynguyen) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance.jsp]: Hi @carolmariaabb 

  I have updated my code, can you review it? 

Thanks.

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**